docker-compose.yml

```diff
@@ -10,7 +10,8 @@ services:
       PROVIDER_CONFIG: |
          {
            "coreProvider": {
-             "maestroToken": "${MAESTRO_API_KEY}"
+             "maestroToken": "${MAESTRO_API_KEY}",
+             "turboSubmit": false
```

Comment:
Isn't turboSubmit false the default? Do we need to add it now, or you just want it to be visible, so people could easily enable it, if they want to?

Reply:
Yes, later.